operators/src/pro/cache/tile_cache.rs

```diff
@@ -477,6 +477,14 @@ impl TileCache {
             .get_mut(&query_id)
             .ok_or(super::error::CacheError::QueryNotFoundInLandingZone)?;
 
+        // TODO: rollback if one fails?
```

Comment:
do you mean one `extend`? this should never happen, right? Otherwise there is a bug in the engine somewhere. So you can remove the TODO(?)

Reply:
[ec74825](https://github.com/geo-engine/geoengine/pull/808/commits/ec74825ab6fe080db6782663ec4ae2274c294c62)